core/controllers/acl_decorators.py

```diff
@@ -1846,9 +1846,11 @@ def test_can_edit_skill(self, skill_id, **kwargs):
         if not self.user_id:
             raise base.UserFacingExceptions.NotLoggedInException
 
-        skill_rights = skill_services.get_skill_rights(skill_id, strict=False)
-        if skill_rights is None:
-            raise base.UserFacingExceptions.PageNotFoundException
+        for single_skill_id in skill_id.split(','):
+            skill_rights = skill_services.get_skill_rights(
```

Comment:
Find a way to do this as get_multi() rather than making N datastore calls.

Also this leaves things open for a malicious URL request that contains a huge number of skills and slows down the rest of the app. You might want to make sure you impose a hard limit on the number of skills before requesting anything from datastore.

Reply:
Done. Will add some tests.

Reply:
Ditto, N datastore calls are still there, right?

Reply:
Oh, I saw `test_can_edit_skills` below. Does that mean this is no longer used?